app/helpers/navigation_helper.rb

```diff
@@ -233,16 +234,14 @@ def add_landing_page_breadcrumb(path)
           session[:breadcrumbs] << { 'display': "#{@page.page_group.name}", 'path': path }
         end
 
-        def add_sub_page_breadcrumb
-          session[:breadcrumbs] << { 'display': "#{@page.title}", 'path': @builder_page_path }
-        end
-
         if @page_slug == 'home'
           empty_breadcrumbs
+        elsif COMMUNITIES.include?(@page.page_group.slug) && @builder_landing_page.exists?
+          empty_breadcrumbs
+          add_landing_page_breadcrumb("/communities/#{params[:page_group_friendly_id]}")
         elsif @builder_landing_page.exists?
           empty_breadcrumbs
           add_landing_page_breadcrumb("/#{params[:page_group_friendly_id]}")
-          add_sub_page_breadcrumb
```

Comment:
Adding to the comment above, we could probably add some code like
```
is_communities_page = COMMUNITIES.include?(@page.page_group.slug)
```
somewhere above the `if/else`, and then do:
```
add_sub_page_breadcrumb unless COMMUNITIES.include?(@page.page_group.slug)
```